piecrust/data/assetor.py

```diff
@@ -92,4 +93,13 @@ def _cacheAssets(self):
         cpi = self._page.app.env.exec_info_stack.current_page_info
         if cpi is not None:
             cpi.render_ctx.current_pass_info.used_assets = True
-
+            
+    def copyAssets(self, page, dest_dir):
```

Comment:
I don't think you need to have `page` passed in here since the current `Assetor` instance already has `self._page` ?